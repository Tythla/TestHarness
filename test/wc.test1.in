test on word count
test