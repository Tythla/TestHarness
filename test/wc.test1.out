       2        5       23 test/wc.test1.in